docs/compendium/substitute.md

```diff
@@ -40,6 +41,8 @@ Request body for a new substitution:
 - `substitutionFiles` - array of file substitutions specified by `base` and `overlay`
   - `base` - name of the file from the base compendium
   - `overlay` - name of the overlay compendium that is exchanged for the original file
+- `metadataHandling` - variable to specify, if the metadata of the base ERC will be adopted (`keepBase` = **keep metadata** of base ERC) or there will be a new extraction of metadata, that will be merged into the metadata of the base ERC (`extractAndMerge` = **extract and merge metadata** for new ERC) or that will not be merged (`extract` = **extract metadata** of new ERC)
```

Comment:
Please use the term "property" instead of variable, to be consistent with rest of docs.